drgn/commands/_builtin/crash/ps.py

```diff
@@ -0,0 +1,294 @@
+
+
+def check_arg_type(arg: Optional[str]) -> Tuple[str, Any]:
+    """
+    Check the filter type of the argument
+    """
+    if arg is not None:
+        try:
+            return ("pid", int(arg, 10))
+        except ValueError:
+            pass
+        try:
+            return ("task", int(arg, 16))
+        except ValueError:
+            return ("comm", arg)
+    else:
+        return ("none", None)
+
+
+@crash_command(
+    description="process information",
+    long_description="display process status information",
+    arguments=(
+        mutually_exclusive_group(
+            argument(
+                "-u",
+                dest="user",
+                action="store_true",
+                default=False,
+                help="display only user threads information",
+            ),
+            argument(
+                "-k",
+                dest="kernel",
+                action="store_true",
+                default=False,
+                help="display only kernel threads information",
+            ),
+        ),
+        mutually_exclusive_group(
+            argument(
+                "-m",
+                dest="last_run",
+                action="store_true",
+                default=False,
+                help="show last run information",
+            ),
+            argument(
+                "-l",
+                dest="last_run_timestamp",
+                action="store_true",
+                default=False,
+                help="show last run information (raw timestamp)",
+            ),
+        ),
+        argument(
+            "-G",
+            dest="group_leader",
+            action="store_true",
+            default=False,
+            help="display only the thread group leader in a thread group",
+        ),
+        argument(
+            "arg",
+            nargs="?",
+            type=check_arg_type,
+            metavar="pid | task | command",
+            help="pid is a process PID. task is hexadecimal task_struct pointer. command is a command name.",
+        ),
+        drgn_argument,
+    ),
+)
+def _crash_cmd_ps(
+    prog: Program, name: str, args: argparse.Namespace, **kwargs: Any
+) -> None:
+
+    if args.drgn:
+        builder = CrashDrgnCodeBuilder(prog)
+        builder.add_from_import("drgn.helpers.linux.pid", "for_each_task")
+        builder.add_from_import("drgn.helpers.common.format", "escape_ascii_string")
+        builder.add_from_import(
+            "drgn.helpers.linux.mm",
+            "task_vsize",
+            "get_task_rss_info",
+            "totalram_pages",
+        )
+        builder.add_from_import(
+            "drgn.helpers.linux.sched", "task_state_to_char", "task_cpu"
+        )
+        if args.last_run or args.last_run_timestamp:
+            builder.add_from_import(
+                "drgn.helpers.linux.sched", "task_since_last_arrival_ns"
+            )
+            builder.add_import("datetime")
+
+        builder.append("total_mem = int(totalram_pages(prog))\n")
```

Comment:
`totalram_pages()` already returns an `int`, so the conversion is unnecessary.
```suggestion
        builder.append("total_mem = totalram_pages(prog)\n")
```